src/main/scala/io/iohk/ethereum/blockchain/sync/fast/FastSync.scala

```diff
@@ -1069,8 +1069,8 @@ class FastSync(
         val bestReceivedBlock = fullBlocks.maxBy(_.number)
         val lastStoredBestBlockNumber = appStateStorage.getBestBlockNumber()
         if (lastStoredBestBlockNumber < bestReceivedBlock.number) {
-          blockchain.saveBestKnownBlocks(bestReceivedBlock.number)
           appStateStorage.putBestBlockNumber(bestReceivedBlock.number).commit()
+          appStateStorage.putBestBlockHash(bestReceivedBlock.hash)
```

Comment:
wouldn't a `.commit()` step here align with the previous line?

Reply:
yes, but I didn't consider at all FastSync. For now we just work regular Sync